docs/source/setup/presentation.rst

```diff
@@ -183,14 +181,14 @@ Apps Tab
 Click the **Apps** tab on the left side of the screen. This will open up a screen that shows applications that you can access through SEPAL. Applications are pre-programmed (typically using R or Python) to perform specific, useful tasks. Applications make use of instances and running an application will use your SEPAL computing resources.
 
 .. image:: ../_images/setup/presentation/apps_interface.png
-   :alt: The Apps interface
+   :alt: The Apps interface.
    :align: center
 
 This will bring up a list of apps you can run in SEPAL. More information about each app is found by clicking on the “i” on the right hand side. Some of the apps include:
 
--   **R Studio:** provides access to R environment where you can run processing scripts and upload data to your SEPAL folder.
--   **Stratified Area Estimator- Design:** tool for creating stratified designs to estimate areas. We will use this tool in Module 4.
--   **Stratified Area Estimator- Analysis:** tool for analyzing the results of your stratified design sampling to estimate areas. We will use this tool in Module 4.
--   **Geo Processing- Beta:** offers a selection of easy-to-use change detection and segmentation tools.
--   **BFAST Explorer:** tool for performing pixel-based time series analysis of Landsat Surface Reflectance data.
+-   **R Studio:** Provides access to the R environment where you can run processing scripts and upload data to your SEPAL folder.
```

Comment:
the name of the best applications have changed significalntly, I need to update this one